.github/workflows/automation.yml

```
@@ -52,13 +52,13 @@ jobs:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - name: scala-mantic
-      run: docker build . --file ./scala/build/Dockerfile.mantic --tag "scala-mantic:$(date +%s)" 
+  # Deprecated - v2024.4.1
+  # build-scala-mantic:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #   - uses: actions/checkout@v4
+  #   - name: scala-mantic
+  #     run: docker build . --file ./scala/build/mantic.Dockerfile --tag "scala-mantic:$(date +%s)" 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)